recipes/libnftnl/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsToolchain, PkgConfigDeps
+from conan.tools.files import copy, get, rmdir, rm
+from conan.tools.layout import basic_layout
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.53.0"
+
+class LibnftnlConan(ConanFile):
+    name = "libnftnl"
+    license = "GPL-2-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://netfilter.org/projects/libnftnl/index.html"
+    description = "Library providing a low-level netlink programming interface (API) to the in-kernel nf_tables subsystem"
+    topics = ("libnftnl", "netlink", "nftables")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    package_type = "library"
```

Comment:
```suggestion
    package_type = "library"
    languages = ["C"]
    implements = ["auto_shared_fpic"]
```
Can drop `config_options()` and `configure()` if `auto_shared_fpic` is added.